tests/system/src/test/java/test/robot/javafx/scene/tableview/TableViewClickOnTroughTest.java

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (c) 2018, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Update the "last touched year" to 2023. Also, unless this is substantially the same as some other class that you copied, you can change the `2018,` to `2022,`.